### ISSUE TITLE
NoReverseMatch at /de/admin/filer/folder/make_folder/ with Django 1.8.1

### ISSUE THREAD

**Issue by rprader** [django-cms/django-filer#526]
I'm getting a NoReverseMatch when I try to create a new folder within the admin-interface. I'm using the newest django version (1.8.1) and also the newest django-filer version (0.9.9).

```
Exception Type: NoReverseMatch at /de/admin/filer/folder/make_folder/
Exception Value: Reverse for 'app_list' with arguments '()' and keyword arguments '{u'app_label': ''}' not found. 1 pattern(s) tried: [u'de/admin/(?P<app_label>filer|auth|redirects|sites)/$']
```

I am also using the newest version of djangocms_admin_style but it doesn't make a difference if I disable this app. Is there a known bug?

**Comment by stefreak** [comment]
When you use master it works for me with Django 1.8. ``pip install -e -e git+git://github.com/stefanfoulis/django-filer.git@master#egg=filer``

Is there a plan to make a new release soon?

**Comment by rprader** [comment]
Oh yes, the current develop-branch also works for me with Django 1.8.1

**Comment by lory87** [comment]
gentle ping @rprader and @stefreak Is this issue fixed with current filer version 1.2.0 ? Thank you for your feedback

**Comment by stefreak** [comment]
@lory87 as the current version tag has been created 15 days ago and my comment is from may 2015, probably yes. But that's a guess.
